src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -25,15 +25,15 @@ export default View.extend({
   initialize () {
     let links = _.resultCtx(this, 'links', this);
     if (this.options.appState.get('showSignoutLink')) {
-      //add cancel/signout link
-      links = links.concat([
+      //add cancel/signout link as the first item of links since its floated right
+      links.unshift(
```

Comment:
<img width="455" alt="Screen Shot 2020-06-15 at 3 43 32 PM" src="https://user-images.githubusercontent.com/23267876/84713699-7978de00-af20-11ea-83af-4475e683944c.png">

This was done to make sure sign out link floats to the right.

Reply:
Would like to chat with Vince to see if we can move `sign out` to left if you have not done so.

Reply:
I did speak to him about this. It seems like for now `sign out` is the only link that they think will be on the right. Other actionable links will be on the left.
@vincentvoong-okta what are your thought on moving sign out link as well to the left ?

Reply:
@nikhilvenkatraman-okta Moving `Sign out` to the left side makes sense and should be below the other options. A note - it should only appear in the appropriate context (e.g. Sign out should only be an option if the user is already signed in. The example above shows Sign out but still asks for the users password)

Reply:
cool, I can update this to be left aligned with sign out as the last option.

![Screen Shot 2020-06-16 at 11 50 17 AM](https://user-images.githubusercontent.com/23267876/84815215-91576d00-afc7-11ea-8b0d-b4d836e9d971.png)

@vincentvoong-okta in most places we show the sign out in the right context. Here the user has entered identifier in the first step before getting to this screen, hence we show the sign out.